didact/src/components/courses/DetailedCourseStyles.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components'
 
 export const DetailedCourseWrapper = styled.div`
-    width: 60%;
+    width : 789px;
```

Comment:
Gave course wrapper a specific width so it doesn't change size when drawer is opened/closed